mri/operators/fourier/non_cartesian.py

```diff
@@ -157,7 +157,7 @@ def adj_op(self, x):
 
 
 class gpuNUFFT:
-    """  GPU implementation of N-D non uniform Fast Fourrier Transform class.
+    """  GPU implementation of N-D Non-uniform Fast Fourrier Transform class.
```

Comment:
(D210) No whitespaces allowed surrounding docstring text
```suggestion
    """ GPU implementation of N-D Non-uniform Fast Fourrier Transform class.
```